app/services/criteria_matcher.py

```
@@ -327,87 +341,19 @@ def _get_macos_last_open_time(file_path: Path) -> Optional[float]:
-                    capture_output=True,
-                    timeout=2,
-                    text=True,
-                )
-                if result.returncode == 0 and result.stdout:
-                    # Extended attribute format may vary, try to parse
-                    # This is a fallback if mdls doesn't work
-                    pass
-            except (subprocess.TimeoutExpired, FileNotFoundError, OSError):
-                pass
-
+            result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)